packages/env/lib/commands/destroy.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * External dependencies
+ */
+const dockerCompose = require( 'docker-compose' );
+const util = require( 'util' );
+const fs = require( 'fs' ).promises;
+const inquirer = require( 'inquirer' );
+
+/**
+ * Promisified dependencies
+ */
+const rimraf = util.promisify( require( 'rimraf' ) );
+
+/**
+ * Internal dependencies
+ */
+const initConfig = require( '../init-config' );
+const stop = require( './stop' );
+
+/**
+ * Destroy the development server.
+ *
+ * @param {Object}  options
+ * @param {Object}  options.spinner A CLI spinner which indicates progress.
+ * @param {boolean} options.debug   True if debug mode is enabled.
+ */
+module.exports = async function destroy( { spinner, debug } ) {
+	const config = await initConfig( { spinner, debug } );
+	const { dockerComposeConfigPath, workDirectoryPath } = config;
+
+	// check installed WordPress.
+	const installed = await fs.readdir( workDirectoryPath );
+	if ( ! installed.length ) {
+		spinner.text = `Not installed WordPress.`;
```

Comment:
```suggestion
		spinner.text = `Could not find any files to remove.`;
```

I think this might be a bit clearer